packages/core/strapi/bin/strapi.js

```diff
@@ -347,7 +352,7 @@ program
         }
 
         await confirmMessage(
-          'The transfer will delete all data in the remote database and media files. Are you sure you want to proceed?',
+          'The transfer will delete all the remote Strapi assets and its database. Are you sure you want to proceed?',
```

Comment:
Just a question that I've just had reading this. What if they are using a third party provider for the assets? They won't be deleted, will they?

Reply:
Yep, non-local providers will still have their assets stored in their respective storage but no Strapi entity will reference them anymore (basically lost entities in a bucket)